### PR TITLE
Add new "stoutgat" keyboard

### PR DESCRIPTION
**Pull request by vattern**
Add new 65% ISO keyboard "stoutgat" with dual encoders

## Description

Add new 65% ISO keyboard "stoutgat" with dual encoders
Uses ATMega32A and dual encoders

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/stoutgat/config.h [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+/* disable these deprecated features by default */
+#ifndef LINK_TIME_OPTIMIZATION_ENABLE
+  #define NO_ACTION_MACRO
+  #define NO_ACTION_FUNCTION
+#endif
+
+
+/* Bootmagic Lite key configuration */
+#define BOOTMAGIC_LITE_ROW 0
+#define BOOTMAGIC_LITE_COLUMN 0
```
Comment (zvecr) [inline keyboards/stoutgat/config.h:56]:
```suggestion
/* disable these deprecated features by default */
#define NO_ACTION_MACRO
#define NO_ACTION_FUNCTION
```

File: keyboards/stoutgat/readme.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# stoutgat
+
+![stoutgat](https://i.imgur.com/CfWa5AG.jpg?1)
+
+A short description of the keyboard/project
```
Comment (zvecr) [inline keyboards/stoutgat/readme.md:5]: This should be filled in

File: keyboards/stoutgat/rules.mk [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE       = no     # USB Nkey Rollover
+BACKLIGHT_ENABLE  = no     # Enable keyboard backlight functionality on B7 by default
+MIDI_ENABLE       = no     # MIDI support
+UNICODE_ENABLE    = no     # Unicode
+BLUETOOTH_ENABLE  = no     # Enable Bluetooth with the Adafruit EZ-Key HID
+AUDIO_ENABLE      = no     # Audio output on port C6
+FAUXCLICKY_ENABLE = no     # Use buzzer to emulate clicky switches
+HD44780_ENABLE    = no     # Enable support for HD44780 based LCDs
+ENCODER_ENABLE    = yes
```
Comment (zvecr) [inline keyboards/stoutgat/rules.mk:40]:
```suggestion
# Bootloader selection
BOOTLOADER = USBasp

# Build Options
#   change yes to no to disable
#
BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
MOUSEKEY_ENABLE = yes       # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = no            # USB Nkey Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
BLUETOOTH_ENABLE = no       # Enable Bluetooth
AUDIO_ENABLE = no           # Audio output
ENCODER_ENABLE  = yes

LAYOUTS = 65_iso
```

File: keyboards/stoutgat/keymaps/default/keymap.c [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+                // backward to go forward instead.
+                if (clockwise) {
+                    tap_code16(C(KC_Z));
+                } else {
+                    tap_code16(C(KC_Y));
+                }
+                break;
+        }
+    } else if (index == 1) {
+        switch (biton32(layer_state)) {
```
Comment (zvecr) [inline keyboards/stoutgat/keymaps/default/keymap.c:56]:
```suggestion
        switch (get_highest_layer(layer_state)) {
```

File: keyboards/stoutgat/keymaps/default/keymap.c [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+   };
+
+
+
+
+#ifdef ENCODER_ENABLE
+void encoder_update_user(uint8_t index, bool clockwise) {
+
+    if (index == 0) {
+        switch (biton32(layer_state)) {
```
Comment (zvecr) [inline keyboards/stoutgat/keymaps/default/keymap.c:36]:
```suggestion
        switch (get_highest_layer(layer_state)) {
```

File: keyboards/stoutgat/info.json [hunk 6]
```diff
@@ -0,0 +1,17 @@
+{
+    "keyboard_name": "stoutgat",
+    "url": "https://github.com/vattern/stoutgat",
+    "maintainer": "Thys de Wet",
+    "width": 16,
+    "height": 5,
+    "layouts": {
+      "LAYOUT_67_iso": {
```
Comment (zvecr) [inline keyboards/stoutgat/info.json:8]:
```suggestion
      "LAYOUT_65_iso": {
```

File: keyboards/stoutgat/info.json [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+             {"x":0, "y":1, "w":1.5}, {"x":1.5, "y":1}, {"x":2.5, "y":1}, {"x":3.5, "y":1}, {"x":4.5, "y":1}, {"x":5.5, "y":1}, {"x":6.5, "y":1}, {"x":7.5, "y":1}, {"x":8.5, "y":1}, {"x":9.5, "y":1}, {"x":10.5, "y":1}, {"x":11.5, "y":1}, {"x":12.5, "y":1}, {"x":15, "y":1},
+             {"x":0, "y":2, "w":1.75}, {"x":1.75, "y":2}, {"x":2.75, "y":2}, {"x":3.75, "y":2}, {"x":4.75, "y":2}, {"x":5.75, "y":2}, {"x":6.75, "y":2}, {"x":7.75, "y":2}, {"x":8.75, "y":2}, {"x":9.75, "y":2}, {"x":10.75, "y":2}, {"x":11.75, "y":2}, {"x":12.75, "y":2}, {"x":13.75, "y":1, "w":1.25, "h":2}, {"x":15, "y":2},
+             {"x":0, "y":3, "w":1.25}, {"x":1.25, "y":3}, {"x":2.25, "y":3}, {"x":3.25, "y":3}, {"x":4.25, "y":3}, {"x":5.25, "y":3}, {"x":6.25, "y":3}, {"x":7.25, "y":3}, {"x":8.25, "y":3}, {"x":9.25, "y":3}, {"x":10.25, "y":3}, {"x":11.25, "y":3}, {"x":12.25, "y":3, "w":1.75}, {"x":14, "y":3}, {"x":15, "y":3},
+             {"x":0, "y":4, "w":1.25}, {"x":1.25, "y":4, "w":1.25}, {"x":2.5, "y":4, "w":1.25}, {"x":3.75, "y":4, "w":6.25}, {"x":10, "y":4}, {"x":11, "y":4}, {"x":12, "y":4}, {"x":13, "y":4}, {"x":14, "y":4}, {"x":15, "y":4}]
+      },
```
Comment (zvecr) [inline keyboards/stoutgat/info.json:15]:
```suggestion
      }
```

File: keyboards/stoutgat/keymaps/default/keymap.c [hunk 8]
```diff
@@ -0,0 +1,75 @@
+#include QMK_KEYBOARD_H
+
+#define _BL 0
+#define _FL 1
+
+
+
+   const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+     [0] = LAYOUT_67_iso(
```
Comment (zvecr) [inline keyboards/stoutgat/keymaps/default/keymap.c:9]:
```suggestion
     [0] = LAYOUT_65_iso(
```

File: keyboards/stoutgat/keymaps/default/keymap.c [hunk 9]
```diff
@@ -0,0 +1,75 @@
...
+         KC_GESC, KC_1,    KC_2,    KC_3,    KC_4,   KC_5,   KC_6,   KC_7,   KC_8,   KC_9,    KC_0,    KC_MINS,  KC_EQL,  KC_BSPC,  KC_GRV,
+         KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,   KC_T,   KC_Y,   KC_U,   KC_I,   KC_O,    KC_P,    KC_LBRC,  KC_RBRC,           KC_DEL,
+         KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,   KC_G,   KC_H,   KC_J,   KC_K,   KC_L,    KC_SCLN, KC_QUOT,  KC_NUHS, KC_ENT,   KC_PGUP,
+         KC_LSFT, KC_NUBS, KC_Z,    KC_X,    KC_C,   KC_V,   KC_B,   KC_N,   KC_M,   KC_COMM, KC_DOT,  KC_SLSH,  KC_RSFT, KC_UP,    KC_PGDN,
+         KC_LCTL, KC_LGUI, KC_LALT,          KC_MPLY,                KC_SPC,  KC_ESC,KC_RALT, MO(1),  KC_RCTL,  KC_LEFT,  KC_DOWN,  KC_RIGHT),
+
+     [1] = LAYOUT_67_iso(
```
Comment (zvecr) [inline keyboards/stoutgat/keymaps/default/keymap.c:16]:
```suggestion
     [1] = LAYOUT_65_iso(
```

File: keyboards/stoutgat/stoutgat.h [hunk 10]
```diff
@@ -0,0 +1,42 @@
...
+
+/* This is a shortcut to help you visually see your layout.
+ *
+ * The first section contains all of the arguments representing the physical
+ * layout of the board and position of the keys.
+ *
+ * The second converts the arguments into a two-dimensional array which
+ * represents the switch matrix.
+ */
+ #define LAYOUT_67_iso( \
```
Comment (zvecr) [inline keyboards/stoutgat/stoutgat.h:30]:
```suggestion
 #define LAYOUT_65_iso( \
```

File: keyboards/stoutgat/stoutgat.h [hunk 11]
```diff
@@ -0,0 +1,42 @@
...
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+#define _x_ KC_NO
+
+#include "quantum.h"
```
Comment (zvecr) [inline keyboards/stoutgat/stoutgat.h:20]:
```suggestion
#include "quantum.h"

#define _x_ KC_NO
```

File: keyboards/stoutgat/rules.mk [hunk 12]
```diff
@@ -0,0 +1,26 @@
...
+COMMAND_ENABLE = no         # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
+AUDIO_ENABLE = no           # Audio output
+ENCODER_ENABLE  = yes
```
Comment (zvecr) [inline keyboards/stoutgat/rules.mk:26]:
```suggestion
ENCODER_ENABLE  = yes

LAYOUTS = 65_iso
```

Comment (fauxpark) [reply keyboards/stoutgat/rules.mk:26]: This is not actually a 65% ISO layout, there are two extra keys for the encoder switches.

File: keyboards/stoutgat/keymaps/default/keymap.c [hunk 13]
```diff
@@ -0,0 +1,75 @@
...
+        /*  tab      Q       W        E        R        T      Y        U      I        O        P        [         ]               delete*/
+         KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, RESET,  KC_TRNS,KC_TRNS,KC_TRNS,KC_INS, KC_TRNS, KC_TRNS, KC_TRNS,  KC_PAUS,           KC_TRNS,
+        /*  caps     A       S        D        F        G      H        J      K        L        ;        '          #     enter     pg up*/
+         KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS, KC_TRNS, KC_TRNS,  KC_INS,  KC_TRNS,  KC_HOME,
+        /* shift     \       Z         X        C       V       B       N      M        ,        .        /        shift     up      pg dn*/
+         KC_LSFT, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_MUTE, KC_TRNS, KC_TRNS, KC_TRNS,  KC_RSFT, KC_VOLU,  KC_END,
+        /* ctrl     win      alt     encoder                  space      encoder        alt      fn       ctrl      left     down     right*/
+         KC_LCTL, KC_LGUI, KC_LALT,  KC_MPLY,                KC_TRNS,  KC_ESC,        KC_RALT, KC_TRNS, KC_RCTL,  KC_TRNS, KC_VOLD,  KC_TRNS)
```
Comment (fauxpark) [inline keyboards/stoutgat/keymaps/default/keymap.c:26]:
```suggestion
                  KC_MPLY,                                                                                                KC_ESC,
         KC_GESC, KC_1,    KC_2,    KC_3,    KC_4,   KC_5,   KC_6,   KC_7,   KC_8,   KC_9,    KC_0,    KC_MINS,  KC_EQL,  KC_BSPC,  KC_GRV,
         KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,   KC_T,   KC_Y,   KC_U,   KC_I,   KC_O,    KC_P,    KC_LBRC,  KC_RBRC,           KC_DEL,
         KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,   KC_G,   KC_H,   KC_J,   KC_K,   KC_L,    KC_SCLN, KC_QUOT,  KC_NUHS, KC_ENT,   KC_PGUP,
         KC_LSFT, KC_NUBS, KC_Z,    KC_X,    KC_C,   KC_V,   KC_B,   KC_N,   KC_M,   KC_COMM, KC_DOT,  KC_SLSH,  KC_RSFT, KC_UP,    KC_PGDN,
         KC_LCTL, KC_LGUI, KC_LALT,                                  KC_SPC,         KC_RALT, MO(1),  KC_RCTL,  KC_LEFT,  KC_DOWN,  KC_RIGHT),

     [1] = LAYOUT_65_iso(
                  KC_TRNS,                                                                                                KC_TRNS,
        /* esc      1        2        3        4       5       6       7       8       9         0        -         =     bkspc       `~  */
         KC_GRV,  KC_F1,   KC_F2,   KC_F3,   KC_F4,  KC_F5,  KC_F6,  KC_F7,  KC_F8,  KC_F9,   KC_F10,  KC_F11,   KC_F12,  KC_DEL,   KC_PSCR,
        /*  tab      Q       W        E        R        T      Y        U      I        O        P        [         ]               delete*/
         KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, RESET,  KC_TRNS,KC_TRNS,KC_TRNS,KC_INS, KC_TRNS, KC_TRNS, KC_TRNS,  KC_PAUS,           KC_TRNS,
        /*  caps     A       S        D        F        G      H        J      K        L        ;        '          #     enter     pg up*/
         KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS, KC_TRNS, KC_TRNS,  KC_INS,  KC_TRNS,  KC_HOME,
        /* shift     \       Z         X        C       V       B       N      M        ,        .        /        shift     up      pg dn*/
         KC_LSFT, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_MUTE, KC_TRNS, KC_TRNS, KC_TRNS,  KC_RSFT, KC_VOLU,  KC_END,
        /* ctrl     win      alt                              space                     alt      fn       ctrl      left     down     right*/
         KC_LCTL, KC_LGUI, KC_LALT,                          KC_TRNS,                 KC_RALT, KC_TRNS, KC_RCTL,  KC_TRNS, KC_VOLD,  KC_TRNS)
```

File: keyboards/stoutgat/stoutgat.h [hunk 14]
```diff
@@ -0,0 +1,42 @@
...
+ *
+ * The second converts the arguments into a two-dimensional array which
+ * represents the switch matrix.
+ */
+#define LAYOUT_65_iso( \
+  K00, K01, K02, K03, K04, K05, K06, K07, K08, K09, K0A, K0B, K0C, K0D, K0E, \
+  K10, K11, K12, K13, K14, K15, K16, K17, K18, K19, K1A, K1B, K1C,      K1E, \
+  K20, K21, K22, K23, K24, K25, K26, K27, K28, K29, K2A, K2B, K2D, K1D, K2E, \
+  K30, K31, K32, K33, K34, K35, K36, K37, K38, K39, K3A, K3B, K3C, K3D, K3E, \
+  K40, K41, K42,      K44,          K46,  K48, K49, K4A, K4B, K4C, K4D, K4E  \
```
Comment (fauxpark) [inline keyboards/stoutgat/stoutgat.h:35]:
```suggestion
       K44,                                                        K48,
  K00, K01, K02, K03, K04, K05, K06, K07, K08, K09, K0A, K0B, K0C, K0D, K0E, \
  K10, K11, K12, K13, K14, K15, K16, K17, K18, K19, K1A, K1B, K1C,      K1E, \
  K20, K21, K22, K23, K24, K25, K26, K27, K28, K29, K2A, K2B, K2D, K1D, K2E, \
  K30, K31, K32, K33, K34, K35, K36, K37, K38, K39, K3A, K3B, K3C, K3D, K3E, \
  K40, K41, K42,                K46,           K49, K4A, K4B, K4C, K4D, K4E  \
```

File: keyboards/stoutgat/rules.mk (hunk 12 above)
Comment (zvecr) [reply keyboards/stoutgat/rules.mk:26]: I have more suggestions, but I had to jump into a meeting.

File: keyboards/stoutgat/stoutgat.h [hunk 15]
```diff
@@ -0,0 +1,43 @@
...
+
+/* This is a shortcut to help you visually see your layout.
+ *
+ * The first section contains all of the arguments representing the physical
+ * layout of the board and position of the keys.
+ *
+ * The second converts the arguments into a two-dimensional array which
+ * represents the switch matrix.
+ */
+#define LAYOUT_65_iso( \
```
Comment (fauxpark) [inline keyboards/stoutgat/stoutgat.h:30]:
```suggestion
#define LAYOUT( \
```

File: keyboards/stoutgat/keymaps/default/keymap.c [hunk 16]
```diff
@@ -0,0 +1,77 @@
...
+   const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+     [0] = LAYOUT_65_iso(
+       KC_MPLY,KC_ESC,
+         KC_GESC, KC_1,    KC_2,    KC_3,    KC_4,   KC_5,   KC_6,   KC_7,   KC_8,   KC_9,    KC_0,    KC_MINS,  KC_EQL,  KC_BSPC,  KC_GRV,
+         KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,   KC_T,   KC_Y,   KC_U,   KC_I,   KC_O,    KC_P,    KC_LBRC,  KC_RBRC,           KC_DEL,
+         KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,   KC_G,   KC_H,   KC_J,   KC_K,   KC_L,    KC_SCLN, KC_QUOT,  KC_NUHS, KC_ENT,   KC_PGUP,
+         KC_LSFT, KC_NUBS, KC_Z,    KC_X,    KC_C,   KC_V,   KC_B,   KC_N,   KC_M,   KC_COMM, KC_DOT,  KC_SLSH,  KC_RSFT, KC_UP,    KC_PGDN,
+         KC_LCTL, KC_LGUI, KC_LALT,                         KC_SPC ,KC_RALT, MO(1),  KC_RCTL,  KC_LEFT,  KC_DOWN,  KC_RIGHT),
+
+     [1] = LAYOUT_65_iso(
```
Comment (fauxpark) [inline keyboards/stoutgat/keymaps/default/keymap.c:17]:
```suggestion
     [0] = LAYOUT(
       KC_MPLY,KC_ESC,
         KC_GESC, KC_1,    KC_2,    KC_3,    KC_4,   KC_5,   KC_6,   KC_7,   KC_8,   KC_9,    KC_0,    KC_MINS,  KC_EQL,  KC_BSPC,  KC_GRV,
         KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,   KC_T,   KC_Y,   KC_U,   KC_I,   KC_O,    KC_P,    KC_LBRC,  KC_RBRC,           KC_DEL,
         KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,   KC_G,   KC_H,   KC_J,   KC_K,   KC_L,    KC_SCLN, KC_QUOT,  KC_NUHS, KC_ENT,   KC_PGUP,
         KC_LSFT, KC_NUBS, KC_Z,    KC_X,    KC_C,   KC_V,   KC_B,   KC_N,   KC_M,   KC_COMM, KC_DOT,  KC_SLSH,  KC_RSFT, KC_UP,    KC_PGDN,
         KC_LCTL, KC_LGUI, KC_LALT,                         KC_SPC ,KC_RALT, MO(1),  KC_RCTL,  KC_LEFT,  KC_DOWN,  KC_RIGHT),

     [1] = LAYOUT(
```

File: keyboards/stoutgat/info.json [hunk 17]
```diff
@@ -0,0 +1,89 @@
+{
+    "keyboard_name": "stoutgat",
+    "url": "https://github.com/vattern/stoutgat",
+    "maintainer": "Thys de Wet",
+    "width": 15,
+    "height": 5,
```
Comment (fauxpark) [inline keyboards/stoutgat/info.json:6]:
```suggestion
    "width": 16,
    "height": 6,
```

File: keyboards/stoutgat/keymaps/default/keymap.c [hunk 18]
```diff
@@ -0,0 +1,77 @@
...
+         KC_GRV,  KC_F1,   KC_F2,   KC_F3,   KC_F4,  KC_F5,  KC_F6,  KC_F7,  KC_F8,  KC_F9,   KC_F10,  KC_F11,   KC_F12,  KC_DEL,   KC_PSCR,
+        /*  tab      Q       W        E        R        T      Y        U      I        O        P        [         ]               delete*/
+         KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, RESET,  KC_TRNS,KC_TRNS,KC_TRNS,KC_INS, KC_TRNS, KC_TRNS, KC_TRNS,  KC_PAUS,           KC_TRNS,
+        /*  caps     A       S        D        F        G      H        J      K        L        ;        '          #     enter     pg up*/
+         KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS, KC_TRNS, KC_TRNS,  KC_INS,  KC_TRNS,  KC_HOME,
+        /* shift     \       Z         X        C       V       B       N      M        ,        .        /        shift     up      pg dn*/
+         KC_LSFT, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_MUTE, KC_TRNS, KC_TRNS, KC_TRNS,  KC_RSFT, KC_VOLU,  KC_END,
+        /* ctrl     win      alt     encoder                  space      encoder        alt      fn       ctrl      left     down     right*/
```
Comment (fauxpark) [inline keyboards/stoutgat/keymaps/default/keymap.c:27]:
```suggestion
        /* ctrl     win      alt                              space                     alt      fn       ctrl      left     down     right*/
```

File: keyboards/stoutgat/info.json [hunk 19]
```diff
@@ -0,0 +1,89 @@
+{
+    "keyboard_name": "stoutgat",
+    "url": "https://github.com/vattern/stoutgat",
+    "maintainer": "Thys de Wet",
```
Comment (noroadsleft) [inline keyboards/stoutgat/info.json:4]: This is meant to be your GitHub username.

```suggestion
    "maintainer": "vattern",
```

File: keyboards/stoutgat/readme.md [hunk 20]
```diff
@@ -0,0 +1,16 @@
+# stoutgat
+
+![stoutgat](https://i.imgur.com/CfWa5AG.jpg?1)
```
Comment (noroadsleft) [inline keyboards/stoutgat/readme.md:3]: Swaps this image for a smaller version.

```suggestion
![stoutgat](https://i.imgur.com/CfWa5AGl.jpg?1)
```

File: keyboards/stoutgat/rules.mk (hunk 12 above)
Comment (vattern) [reply keyboards/stoutgat/rules.mk:26]: I'm still new to the whole layout bit, how would the community layouts get applied?

Comment (drashna) [reply keyboards/stoutgat/rules.mk:26]: @vattern https://docs.qmk.fm/#/feature_layouts

This page should cover most of how to use the community layout feature, and how ti implement it.

The big thing is that the layout needs to match what is in /layouts/default/ or /layouts/community/ for the specific layout in question.

Comment (vattern) [reply keyboards/stoutgat/rules.mk:26]: I've added 65_iso and 65_ansi

File: keyboards/stoutgat/config.h [hunk 21]
```diff
@@ -0,0 +1,44 @@
...
+#define PRODUCT_ID 0x7811
+#define DEVICE_VER 0x0001
+#define MANUFACTURER TKW
+#define PRODUCT stoutgat
+
+/* key matrix size */
+#define MATRIX_ROWS 5
+#define MATRIX_COLS 15
+#define TAP_CODE_DELAY 10
+#define ENCODER_RESOLUTION 4
```
Comment (fauxpark) [inline keyboards/stoutgat/config.h:31]:
```suggestion
```

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**